### DISCUSSION TITLE
Cannot read properties undefined with wrap

### DISCUSSION THREAD

**Discussion by cotegeo** [maplibre/maplibre-gl-js#3137] [Q&A]
Hello,

I have a problem with a Maplibre map. I have this error in console, but I don't know this message means exactly. 
I have a function which filters on one or several towns, according to the customer. At the opening of the app, app zooms on these towns `map.fitBounds(townsbds)`. But the app failed after few seconds.

![Capture d’écran du 2023-09-29 12-43-57](https://github.com/maplibre/maplibre-gl-js/assets/143707295/646df5fc-3f69-44d3-b1f9-5996e74c89f3)

![Capture d’écran du 2023-09-29 12-56-35](https://github.com/maplibre/maplibre-gl-js/assets/143707295/420b3d2e-fd26-48ad-a03d-2d3411e66a68)

I work with cadastral datas (geometry multipolygon, EPSG:2154). I have an API where geometry is reprojected in EPSG:3857 inside a vector tile.

I have another app where I haven't this problem. My data is geometry point, EPSG:4326.

Message from `terrain.ts` seems to be about projections. 
Is it possible that this comes from projections ? Or geometry multi ?

Thanks

Sylvain BECQUAERT

MAJ 29/9 13:59 = The problem seems to be about DEM. I have Maplibre-DEM on my app. But this doesn't explain why that not working on an app but it's ok on the other app.

**Comment by HarelM** [comment]
It might be that lngLat is not a class but only a type with the info and the wrap method is missing, IDK, I'll need to see a reproduction to better understand what went wrong.
This code is used to calculate the minimum and maximum elevation in the tile to better calculate near and far Z.
this code was introduced lately, it might be that the differences between the apps in maplibre's version...
